src/ucp/rndv/proto_rndv.c

```diff
@@ -636,6 +637,7 @@ void ucp_proto_rndv_bulk_query(const ucp_proto_query_params_t *params,
     attr->max_msg_length = SIZE_MAX;
     attr->is_estimation  = 0;
     ucp_proto_multi_query_config(&multi_query_params, attr);
+    attr->lane_map |= UCS_BIT(rpriv->super.lane);
```

Comment:
maybe move it before line 639?

Reply:
It goes after `ucp_proto_multi_query_config` in order to store `UCS_BIT(rpriv->super.lane)` along with chages made in `ucp_proto_multi_query_config`.